Update admin-key-concepts.md (#529)
MicrosoftDocs/OfficeDocs-O365ITPro: yammer/get-started-with-yammer/admin-key-concepts.md

    
- If you do not choose to enforce Office 365 identity, users who are on the same email domain as your Yammer network but do not have Office 365 licenses can join the network by creating a Yammer account at http://www.yammer.com.
    

>     There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users can be made admins, create groups, and use all Yammer Basic features. If they are made admins, they have to get to the Yammer admin center by clicking the **Settings** icon in Yammer, and selecting **Network Admin.**

Reviewer: ```suggestion
    There are some differences in how you manage these users: their profile won't be automatically filled in, and you can't block them by removing their Yammer license. These users can be made admins, create groups, and use all basic Yammer features. If they are made admins, they have to get to the Yammer admin center by clicking the **Settings** icon in Yammer, and selecting **Network Admin**.
```

> - Users will need to have an enterprise license to use Yammer enterprise features.

Reviewer: ```suggestion
- Users need a Yammer Enterprise license to use [Yammer Enterprise features](https://docs.microsoft.com/yammer/get-started-with-yammer/compare-options-for-a-free-trial).
```
